source/declarations.tex

```diff
@@ -3918,14 +3918,12 @@
 }
 \end{codeblock}
 
-\pnum
 The \tcode{carries_dependency} attribute on function \tcode{f} means that the
```

Comment:
(To be clear, these *are* still paragraph breaks, they're just not numbered. I'd like to call these "underhanded paragraphs". We do this in a few places.)